src/analysis/Analyses.ts

```diff
@@ -331,12 +332,12 @@ const AnalysisCard = ({
     [getFileName(name)]
   );
 
-  const toolIconSrc: string = Utils.switchCase(
-    tool,
-    [runtimeToolLabels.Jupyter, () => jupyterLogo],
-    [runtimeToolLabels.RStudio, () => rstudioSquareLogo],
-    [runtimeToolLabels.JupyterLab, () => jupyterLogo]
-  );
+  const toolLogos: Record<RuntimeToolLabel, string> = {
+    [runtimeToolLabels.Jupyter]: jupyterLogo,
+    [runtimeToolLabels.RStudio]: rstudioSquareLogo,
+    [runtimeToolLabels.JupyterLab]: jupyterLogo,
+  };
+  const toolIconSrc: string | undefined = toolLogos[tool];
```

Comment:
This can actually be undefined since `tool` is a `ToolLabel`, not necessarily a `RuntimeToolLabel`.